lib/fortnox/api/request_handling.rb

```diff
@@ -16,6 +16,13 @@ def raise_api_error(error, response)
       def validate_response(response)
         return if response.code == 200
 
+        if response.headers['content-type'].start_with?('text/html')
```

Comment:
This also commonly happen when their API is down etc. For some reason you get a default 500 HTML page as response when they are down ...